env-check.sh

```diff
@@ -24,12 +27,22 @@ func_check_python_pkg(){
     fi
 }
 
+func_check_file(){
+    if [ -e $1 ]; then
+        return
+    else
+        out_str=$out_str"\nOptional: Enable dynamic debug logs on \e[31m $1 \e[0m file\n\tFor example,\n\toptions snd_sof dyndbg=+p\n\toptions snd_sof_pci dyndbg=+p"
```

Comment:
it's not optional for CI? Do we have a mechanism to be strict with env-check.sh and make it really picky when running from a CI environment (maybe extra parameter?)

Reply:
yes, this is not optional for CI. We will mandate same sof-dyndbg.conf to all CI devices. When a build is deployed to CI device, it will be downloaded.
Separate discussion is started optimal verbosity for SOF CI.